web/packages/teleport/src/components/TdpClientCanvas/TdpClientCanvas.tsx

```diff
@@ -298,20 +299,25 @@ export default function TdpClientCanvas(props: Props) {
 
   // Call init after all listeners have been registered
   useEffect(() => {
-    if (tdpCli && tdpCliInit) {
-      tdpCli.init();
+    if (tdpCli && tdpCliConnect) {
+      tdpCli.connect(tdpCliScreenSpec);
       return () => {
         tdpCli.shutdown();
       };
     }
-  }, [tdpCli, tdpCliInit]);
+  }, [tdpCli, tdpCliConnect]);
 
   return <canvas style={{ ...style }} ref={canvasRef} />;
 }
 
 export type Props = {
-  tdpCli?: TdpClient;
-  tdpCliInit?: boolean;
+  tdpCli: TdpClient;
```

Comment:
```suggestion
  tdpClient: TdpClient;
```

`client` is so much easier to comprehend than `cli`, I don't think we need to save 3 keystrokes.

Reply:
See fcb93f165119a955b9024d900b84a9236556abb9